requirements.txt

```diff
@@ -43,6 +43,7 @@ simplejson==3.6.5
 six==1.9.0
 thriftpy==0.1.15
 tron==0.6.1.1
+ujson==1.35
```

Comment:
yup, doesn't look like it has any dependencies https://github.com/esnme/ultrajson/blob/master/setup.py